ArdalisRating/samplePolicies.json.txt

```diff
@@ -0,0 +1,62 @@
+﻿/* Valid Auto Policy */
+{
+  "type": "Auto",
+  "make": "BMW"
+}
+
+/* No Make */
+{
+  "type": "Auto"
+}
+
+/*
+LAND Samples
+*/
+
+/* Valid Land Policy */
+{
+  "type": "Land",
+  "bondAmount": "1000000",
```

Comment:
Your program will not be working directly with the file, you can write all objects to the json array in this file and you will be able deserialize such file to list of Rating objects and work with each item

Reply:
Actually, this file is deserialised and data is received from it.